package.json

```diff
@@ -44,21 +44,22 @@
   },
   "dependencies": {
     "babel-runtime": "^6.26.0",
-    "invariant": "^2.2.4",
     "memoize-one": "^3.1.1",
     "prop-types": "^15.6.0",
     "raf-schd": "^2.1.1",
     "react-motion": "^0.5.2",
     "react-redux": "^5.0.6",
     "redux": "^3.7.2",
     "redux-thunk": "^2.2.0",
-    "reselect": "^3.0.1"
+    "reselect": "^3.0.1",
+    "tiny-invariant": "^0.0.3"
```

Comment:
right now i have not figured out how to get our rollup build to tree shake the env conditional insight of this module. However, i am happy to solve that later